packages/foam-vscode/src/features/filtered-resources.ts

```diff
@@ -0,0 +1,252 @@
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as fs from 'fs';
+import {
+  IDataStore,
+  URI,
+  FoamWorkspace,
+  Resource,
+  isAttachment,
+  isPlaceholder,
+  getTitle,
+} from 'foam-core';
+import micromatch from 'micromatch';
+import {
+  FilteredResourcesConfig,
+  FilteredResourcesConfigGroupBy,
+} from '../settings';
+import { getContainsTooltip, getNoteTooltip } from '../utils';
+import { dirname, join } from 'path';
+import { FoamFeature } from '../types';
+import { commands } from 'vscode';
+
+const feature: FoamFeature = {
+  activate: (context: vscode.ExtensionContext) => {
+    context.subscriptions.push(
+      commands.registerCommand(
```

Comment:
I understand you are making this a feature to be able to register this command.
But this command in reality is not related to the feature, it's just used by it.
I would move this into a `feature/utility-commands.ts` file